clock_exercise/main.cpp

```diff
@@ -2,6 +2,8 @@
 #include <QQmlApplicationEngine>
 #include <QQmlContext>
 #include "clockmanager.h"
+#include "alarmitemmodel.h"
+#include "alarmitemdata.h"
```

Comment:
Siamo sicuri che ci servano questi include? Non stiamo usando da nessuna parte queste classi qui in main.cpp, giusto?